rel-eng/releasers.conf

```diff
@@ -29,6 +38,10 @@ releaser = tito.release.KojiReleaser
 autobuild_tags = foreman-client-nightly-rhel5 foreman-client-nightly-rhel6 foreman-client-nightly-rhel7 foreman-client-nightly-fedora27 foreman-client-nightly-fedora28
 builder = tito.builder.FetchBuilder
 
+[koji-foreman-client-nightly]
```

Comment:
This releaser seems to be defined exactly the same as `koji-foreman-client`. What is the benefit of re-defining under a new name?

Reply:
whooooooops!